### DISCUSSION TITLE
QR auth flow

### DISCUSSION THREAD

**Discussion by idkroff** [gotd/td#1147] [Q&A]
Is there any example on how to use QR code auth flow to authorize into user account?

With 2fa?
Getting 401: SESSION_PASSWORD_REQUIRED

**Comment by floodwm** [comment]
> Is there any example on how to use QR code auth flow to authorize into user account?
> 
> With 2fa? Getting 401: SESSION_PASSWORD_REQUIRED

Hi, did you manage to implement it? Would you share an example?
